pkg/cmd/volume/rm.go

```diff
@@ -33,17 +33,10 @@ import (
 )
 
 func Remove(ctx context.Context, client *containerd.Client, volumes []string, options types.VolumeRemoveOptions) error {
```

Comment:
Thinking more about this, the entire section starting with client.Containers should be inside the locked segment.
Will amend.